packages/icons/src/HarnessIcons.ts

```diff
@@ -3355,6 +3359,7 @@ const HarnessIcons: KVO<ElementType> = {
   github: Github,
   'gitlab-selected': GitlabSelected,
   'gitlab-unselected': GitlabUnselected,
+  'gitlab-viewbox': GitlabViewbox,
```

Comment:
This icon is already available with name gitlab.

Reply:
Existing icon is not having viewbox which causing issues for scaling.
Not replaced as it might cause regression.
1st point mentioned in below doc for more information
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices